src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -146,6 +162,22 @@ protected internal override async Task<HttpResponseMessage> SendAsync(HttpReques
 
                 requestObject.SetObjectProperty("method", request.Method.Method);
 
+                // Only set if property was specifically modified and is not default value
+                if (_isAllowAutoRedirectTouched)
```

Comment:
Does wasm have multi-threading? How strong/weak its memory model is? Could we get stale reads of either of these fields?

Reply:
no multi threading